lib/device.js

```diff
@@ -104,10 +104,26 @@ function device(_system, panel) {
 		}
 	};
 
+	system.on('lockoutall', function () {
```

Comment:
Everytime you add a listener on system, you need to make sure the function is a reference, and that you unload that reference in the function unload(). If not, this callback will continue to try to run, even after the device is removed. This is the reason for MrLazerface's problems. If you remove a stream deck, and add a new, your callbacks will try to modify a device that should have been removed and garbage-collected. Look at how "elgato_click" has been implemented in this file.